pallet-chainlink-feed/src/benchmarking.rs

```diff
@@ -522,21 +522,13 @@ benchmarks! {
 			vec![(oracle.clone(), admin)],
 		));
 		let feed = Zero::zero();
-		let answer: T::Value = 42u8.into();
-		let rounds: RoundId = 4;
-		let fund_account = T::PalletId::get().into_account();
-		T::Currency::make_free_balance_be(&fund_account, Zero::zero());
-		for round in 1..(rounds + 1) {
-			assert_is_ok(ChainlinkFeed::<T>::submit(RawOrigin::Signed(oracle.clone()).into(), feed, round, answer));
-		}
-		let rounds: BalanceOf<T> = rounds.into();
-		let debt: BalanceOf<T> = rounds * payment;
-		assert_eq!(Debt::<T>::get(), debt);
-		T::Currency::make_free_balance_be(&fund_account, payment + payment);
+		let payment = 1_000;
+		T::Currency::make_free_balance_be(&caller, payment);
+		ChainlinkFeed::<T>::fund_feed(	RawOrigin::Signed(caller.clone()).into(),feed, payment);
 	}: _(RawOrigin::Signed(caller.clone()), payment)
 	verify {
 		assert_eq!(T::Currency::free_balance(&fund_account), payment);
```

Comment:
`fund_account` not found on my machine


run: `cargo build --features runtime-benchmarks` under `substrate-node-example`